queries/special.py

```diff
@@ -635,20 +644,34 @@ def _play_film(qs: str, q: Query) -> AnswerType:
     "ert þú að hlusta": _YES,
     "ertu að hlusta á mig": _YES,
     "ert þú að hlusta á mig": _YES,
+    "ertu að hlusta á okkur": _YES,
```

Comment:
Hmm. Could cause misunderstanding. Perhaps a better answer would be "Já, ef þú segir hæ Embla."